spec/models/post_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+RSpec.describe Post, type: :model do
+  # tests go here
+  before :each do
+    @user = User.new(name: 'Madix', post_counter: 0)
+  end
+
+  it 'Title should always be present' do
+    post = Post.new(author: @user, Title: 'Some Test', comments_counter: 0, likes_counter: 0)
+    expect(post).to be_valid
+  end
+
+  it 'Title should be a maximum of 250 chars' do
+    post = Post.new(author: @user, Title: 'A' * 251, comments_counter: 0, likes_counter: 0)
+    expect(post).to_not be_valid
```

Comment:
- [ ] All custom methods in the User, Post, Comment, and Like models should be tested. Kindly do so. You have methods such as `recent_post` in the user model, `update_counter`, and `recent_comments` methods in the Post model to mention a few. Kindly ensure you add tests for all your models.

Reply:
- [ ] Also please do not forget to add tests for CUSTOM methods in the Like and Comment models as per the requirement.